Dockerfile-flask

```diff
@@ -5,7 +5,10 @@ EXPOSE 6000
 
 # set environment variables
 ENV FLASK_APP=yaptide.application
-WORKDIR /usr/local/app
+WORKDIR /home
+
+RUN python -m venv /home/venv && chmod -R 777 /home/venv
+ENV PATH="/home/venv/bin:$PATH"
```

Comment:
Is this really the only way to install pip packages in venv ?
have you tried any other ways, like `pip install --user` ?

Reply:
`pip install --user` seems to not work, because open shift containers don't instantiate home directory for user, so `USER_SITE` does not exist. I will try to change the owner of the venv directory to the container user and I think it will be the best I can do.

Reply:
OK, thanks for explanation !